gpkitmodels/SP/SimPleAC/SimPleAC.py

```diff
@@ -0,0 +1,94 @@
+from gpkit import Variable, Model, SignomialsEnabled, VarKey, units
+import numpy as np
+import matplotlib.pyplot as plt
+
+def SimPleAC():
```

Comment:
maybe make this inherit from Model instead of being a function?

Reply:
I had some weird bugs with that recently, don't know why. I am planning on changing it.

Reply:
I am not planning on merging this until I clean up the write-up.

Reply:
:+1: